### PR TITLE
AV: Add "Duration" to durationSpoken text

### LINKED ISSUE #4745: AV: Duration in psammead-play-button doesn't follow screen reader UX
**Describe the bug**
When using any screen reader with our media player, the play button duration is currently being read out literally e.g. `'3,11' (Three comma eleven)` which doesn't follow our screen reader UX.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.test.bbc.co.uk/news/articles/c3wmq4d1y3wo
2. Use any screen reader listed in our ['Assistive Technology'](https://bbc.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology)
3. Navigate to the media play button with the screen reader

**Expected behavior**
We can have it say `'Duration 3,11'`. Note, 'Duration' should be translated. And that ideally this should say '3 minutes, 11 seconds'.

**Screenshots**

**Testing notes**

- [x] This bug fix is expected to need manual testing.

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Resolves #4745

**Overall change:** To improve screen reader UX for when the duration is read out, the string "Duration" has been added before the duration is read out to visually hidden text.

**Code changes:**

- Added durationSpokenPrefix string to durationSpoken text.
- Implemented translations for `Duration` for services that have it available in their config.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Review by RayNjeri — APPROVED** [review]

**Review by AlistairGempf — APPROVED** [review]
Looks good

**Comment by HarveyPeachey** [comment]
**Dev/A11y Swarm**

-https://bbc.github.io/accessibility-news-and-you/accessibility-news-and-testers

_Screen Readers (IE11) - Harvey:_ 
- [x] ZoomText Magnifier/Reader
Dragon NaturallySpeaking - N/A
- [x] JAWS
- [x] Read&Write
- [x] NVDA

_Mac, iOS, and Android - Ray:_ 
- [x] VoiceOver iOS (iPhone/iPad)
- [x] VoiceOver OS (Mac)
- [x] TalkBack (Chrome on Android)

File: src/app/containers/MediaPlayer/index.jsx [hunk 1]
```diff
@@ -63,11 +63,14 @@ const MediaPlayerContainer = ({
     aresMediaBlock,
   );
   const duration = moment.duration(rawDuration, 'seconds');
+  const durationSpokenPrefix = translations.media.duration
```
Comment (rhenshaw56) [inline src/app/containers/MediaPlayer/index.jsx]:
```suggestion
  const durationSpokenPrefix = pathOr('Duration', ['media', 'duration'], translations);
```

**Review by rhenshaw56 — COMMENTED** [review]
Looks good. Just added a suggestion

File: src/app/containers/MediaPlayer/index.jsx (hunk 1 above)
Comment (HarveyPeachey) [reply src/app/containers/MediaPlayer/index.jsx]: Ahh okay so this a fancier way of doing a ternary expression? 

```
const durationSpokenPrefix = translations.media.duration
    ? translations.media.duration
    : 'Duration';
```

**Review by j-pendlebury — APPROVED** [review]

**Review by jamesdonoh — APPROVED** [review]
Nice, just one question:
> Implemented translations for Duration for services that have it available in their config.

I can't see any of these, were they removed? Presumably we've checked that the relevant config key is used if present?

**Comment by HarveyPeachey** [comment]
> Nice, just one question:
> 
> > Implemented translations for Duration for services that have it available in their config.
> 
> I can't see any of these, were they removed? Presumably we've checked that the relevant config key is used if present?

The translations we have available have been added already. I have checked that the relevant config key is used if present.

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**